config/openGemini.singlenode.conf

```diff
@@ -9,6 +9,7 @@
 
 [http]
   bind-address = "127.0.0.1:8086"
+  domain = "localhost"

```

Comment:
`domain` is only used in cloud-native scene

Reply:
I have fixed.